test/cppsuite/test_harness/test.h

```diff
@@ -112,29 +113,30 @@ class test : public database_operation {
         int64_t cache_size_mb, duration_seconds;
         bool enable_logging, statistics_logging;
         configuration *statistics_config;
-        std::string statistics_type;
+        std::string debug_mode, statistics_type;
```

Comment:
Where is `debug_mode` used ?

Reply:
Good catch, accidently added that from previous iteration of this change.